python/ray/experimental/sgd/sgd_worker.py

```diff
@@ -0,0 +1,254 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import time
+
+import pyarrow.plasma as plasma
+import tensorflow as tf
+
+import ray
+from ray.experimental.sgd.util import fetch, run_timeline, warmup
+from ray.experimental.sgd.modified_allreduce import sum_gradients_all_reduce, \
+    unpack_small_tensors
+
+logger = logging.getLogger(__name__)
```

Comment:
A unified logger module is better if we want to set a unified logging level, logging format. Currently this kind of logger is related to `logging` module, which may be interleaved with user's logging configuration and cause trouble for logging.